src/pretix/plugins/stripe/payment.py

```diff
@@ -950,6 +1048,109 @@ def payment_presale_render(self, payment: OrderPayment) -> str:
                f'{_("expires {month}/{year}").format(month=card.get("exp_month"), year=card.get("exp_year"))}'
 
 
+class StripeSEPADirectDebit(StripePaymentIntentMethod):
+    identifier = 'stripe_sepa_debit'
+    verbose_name = _('SEPA Debit via Stripe')
+    public_name = _('SEPA Debit')
+    method = 'sepa_debit'
+
+    def payment_form_render(self, request: HttpRequest, total: Decimal, order: Order=None) -> str:
+        cs = cart_session(request)
+
+        template = get_template('pretixplugins/stripe/checkout_payment_form_sepadirectdebit.html')
+        ctx = {
+            'request': request,
+            'event': self.event,
+            'settings': self.settings,
+            'form': self.payment_form(request),
+            'email': order.email if order else cs.get('email', '')
+        }
+        return template.render(ctx)
+
+    @property
+    def payment_form_fields(self):
+        return OrderedDict(
+            [
+                ('accountname',
```

Comment:
Can we find a way to prefill this and all of the other fields from the invoice address from the cart or order?